core/src/main/java/com/yandex/yoctodb/util/UnsignedByteArrays.java

```diff
@@ -82,6 +82,12 @@ public static UnsignedByteArray from(final int i) {
         return from(Ints.toByteArray(i ^ Integer.MIN_VALUE));
     }
 
+    @NotNull
+    public static UnsignedByteArray from(final char c) {
+        final int ci = c ^ Character.MIN_VALUE;
+        return from(Chars.toByteArray(Chars.checkedCast(ci)));
```

Comment:
So, since XOR is redundant, you can simplify this part and throw away `checkedCast`

Reply:
Agree, fixed.